Fix(GW): Add more info about license generation (#4939)
Kong/developer.konghq.com: app/_gateway_entities/license.md

## License reports

> A license report contains information about your {{site.base_gateway}} deployment, including license usage and deployment information. Reports are generated manually on demand with the Admin API and aren't generated automatically or on a schedule. The report doesn't send data to Kong servers.

Reviewer: ```suggestion
A license report contains information about your {{site.base_gateway}} deployment, including license usage and deployment information. 
Reports are generated manually on demand with the Admin API, you can't generate them automatically or on a schedule. 
The report doesn't send data to Kong servers.
```

You can share the report with Kong Support to perform a health-check analysis of product usage and overall deployment performance to ensure your organization is optimized with the best license and deployment plan for your needs.

To generate a license report, send a `GET` request to the [`/license/report` endpoint](/api/gateway/admin-ee/#/operations/get-license-report):

```sh
curl http://localhost:8001/license/report
```

The response contains the following fields:

<!--vale off-->
{% table %}
columns:
  - title: Field
    key: field
  - title: Description
    key: description
rows:
  - field: "`checksum`"
    description: SHA-256 digest of the full report contents.
  - field: "`timestamp`"
    description: Unix timestamp of when the report was generated.
  - field: "`kong_version`"
    description: The version of {{site.base_gateway}} running on this node.
  - field: "`db_version`"
    description: The database engine and version in use.
  - field: "`license`"
    description: "Details about the active license: `license_key` and `license_expiration_date`."
  - field: "`deployment_info`"

>     description: "The deployment topology type (`traditional`, `hybrid`, or `dbless`). In hybrid mode, also includes `connected_dp_count`: the number of data planes currently connected to the control plane."

Reviewer: ```suggestion
    description: "The deployment topology type (`traditional`, `hybrid`, or `dbless`). In hybrid mode, this also includes `connected_dp_count`, the number of data planes currently connected to the control plane."
```

  - field: "`system_info`"
    description: "Information about the node: `hostname`, `cores` (number of online processors), and `uname` (OS and architecture)."
  - field: "`services_count`"

>     description: Total number of services configured in this deployment.

Reviewer: ```suggestion
    description: Total number of Services configured in this deployment.
```

  - field: "`routes_count`"

>     description: Total number of routes configured in this deployment.

Reviewer: ```suggestion
    description: Total number of Routes configured in this deployment.
```

  - field: "`consumers_count`"

>     description: Total number of consumers configured in this deployment.

Reviewer: ```suggestion
    description: Total number of Consumers configured in this deployment.
```

  - field: "`workspaces_count`"

>     description: Total number of workspaces in this deployment.

Reviewer: ```suggestion
    description: Total number of Workspaces in this deployment.
```

  - field: "`rbac_users`"

>     description: Total number of RBAC users configured.

Reviewer: ```suggestion
    description: Total number of configured RBAC users.
```

  - field: "`plugins_count`"

>     description: "Plugin usage broken down into `tiers` (`free`, `enterprise`, `custom`), `unique_route_kafkas` (unique Kafka broker addresses on service-less routes), and `unique_route_lambdas` (unique Lambda function names on service-less routes)."

Reviewer: ```suggestion
    description: "Plugin usage broken down into `tiers` (`free`, `enterprise`, `custom`), `unique_route_kafkas` (unique Kafka broker addresses on Service-less Routes), and `unique_route_lambdas` (unique Lambda function names on Service-less Routes)."
```
